### PR TITLE
[FIXED JENKINS-28154] Confirm fix of problem with operators

### PR DESCRIPTION
**Pull request by jglick**
Simply a mergeable (and passing) version of #17.

@reviewbybees

### CODE REVIEW

File: src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java [hunk 1]
```diff
@@ -224,7 +227,7 @@ public void setProp2(String prop2) {
             this._prop2 = prop2;
         }
     }
-    
+
```
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java]: uhm, you know :)

**Comment by oleg-nenashev** [comment]
:bee:

File: src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java (hunk 1 above)
Comment (jglick) [inline src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/groovy/SandboxInterceptorTest.java]: Complain to @olivergondza. :-)

**Comment by olivergondza** [comment]
Thanks, I have not noticed the merge ...

**Pull request merged**